react-commenting.js

```diff
@@ -30,19 +30,18 @@ class Board extends Component{
   }
 
   gamePlay = (index) => {
-    // 3) Here:
-    const { gameBoard, currentPlayer, winner, clickCount } = this.state
-    // 4) Here:
+    // 3) Here: here you put in the arguments that the function is going to take 
```

Comment:
This is true, but we are getting the arguments by destructuring the variables out of state.